source/core/src/main/com/csse3200/game/screens/MainGameScreen.java

```diff
@@ -144,4 +173,65 @@ private void createUI() {
 
     ServiceLocator.getEntityService().register(ui);
   }
+
+  public void addOverlay(OverlayType overlayType){
```

Comment:
These methods dont have any test coverage, and also lack documentation.

Reply:
documentation added

Reply:
Been visually tested.